src/lamp_py/performance_manager/alerts.py

```diff
@@ -307,7 +307,7 @@ def extract_start_end(
         periods: Optional[List[Dict[str, int]]]
     ) -> Tuple[int | None, int | None]:
         """small lambda for extracting start and end timestamps"""
-        if periods is None:
+        if periods is None or len(periods) == 0:
```

Comment:
handle the case where periods is an empty list.

i'm not entirely sure why that should be an output from ingestion but it is.